src/frontend/web_application/src/styles/object/o-clickable.scss

```diff
@@ -57,7 +57,7 @@
 
 @mixin o-clickable--expanded {
   display: block;
-  line-height: inherit;
+  // line-height: inherit;
```

Comment:
yes it is a bit weird but useful when a button has not a regular height (callback buttons for instance).
This permit to vertically align text within large buttons

So ok to remove if you have a better solution